components/homme/test/reg_test/run_tests/test-list.cmake

```diff
@@ -106,7 +106,9 @@ IF (BUILD_HOMME_THETA_KOKKOS)
     thetah-nhgw-slice-kokkos.cmake
     thetanh-nhgw-slice-kokkos.cmake
     thetanh-moist-bubble-sl.cmake
-    thetanh-moist-bubble-sl-kokkos.cmake)
+    thetanh-moist-bubble-sl-kokkos.cmake
+    thetanh-moist-bubble-sl-pg2.cmake
+    thetanh-moist-bubble-sl-pg2-kokkos.cmake)
```

Comment:
it seems pg2 tests in this list are planar, can we use planar in their names?

Reply:
I can, but please note I'm following the naming scheme that I believe you established.